### PR TITLE
Add norm asserts for ssa iso notebooks

### PR DESCRIPTION
**Pull request by jkwashbourne**
As discussed on slack, this PR adds assertion of norms for computed wavefields.

### CODE REVIEW

**Comment by mloubout** [comment]
Nice thanks, notebook should now be added to the notebooks CI https://github.com/devitocodes/devito/blob/master/.github/workflows/tutorials.yml

File: .github/workflows/tutorials.yml [hunk 1]
```diff
@@ -93,6 +93,13 @@ jobs:
         $RUN_CMD py.test --nbval --cov . --cov-config=.coveragerc --cov-report=xml:seis_coverage.xml $SKIP examples/seismic/tutorials/
         $RUN_CMD py.test --nbval --cov . --cov-config=.coveragerc --cov-report=xml:acc_coverage.xml $SKIP examples/seismic/acoustic/accuracy.ipynb
 
+    - name: Self-adjoint notebooks
+      run: |
+        $RUN_CMD py.test --nbval --cov . --cov-config=.coveragerc --cov-report=xml:sa_coverage.xml $SKIP examples/seismic/self_adjoint/
```
Comment (mloubout) [inline .github/workflows/tutorials.yml:98]: just `$RUN_CMD py.test --nbval --cov . --cov-config=.coveragerc --cov-report=xml:acc_coverage.xml $SKIP examples/self_adjoint/` should be enough

Comment (jkwashbourne) [reply .github/workflows/tutorials.yml:98]: acc_coverage?
why not sa_coverage?

Comment (mloubout) [reply .github/workflows/tutorials.yml:98]: No I mean you only need one command not one per notebook.

Comment (mloubout) [reply .github/workflows/tutorials.yml:98]: so yes just:
```
- name: Self-adjoint notebooks
run: |
        $RUN_CMD py.test --nbval --cov . --cov-config=.coveragerc --cov-report=xml:sa_coverage.xml $SKIP examples/seismic/self_adjoint/
```

Comment (jkwashbourne) [reply .github/workflows/tutorials.yml:98]: got it

**Comment by jkwashbourne** [comment]
@mloubout 
How can I tell if I need to rebase on master for this?

**Pull request merged**